src/mono/wasm/Makefile

```diff
@@ -72,7 +72,7 @@ MONO_LIBS = \
 	$(ICU_LIBDIR)/libicui18n.a
 
 EMCC_DEBUG_FLAGS =-g -Os -s -DDEBUG=1
-EMCC_RELEASE_FLAGS=-Oz
+EMCC_RELEASE_FLAGS=-O2
```

Comment:
Is this needed on `main` too?

Reply:
It could go there but it isn't used as part of the normal build

Reply:
We don't use these flags in main anymore. I should open PR to remove this variable.